docs/containers-transports.5.md

```diff
@@ -0,0 +1,53 @@
+% CONTAINERS-TRANSPORTS(5) Containers Transports Man Page
+% Valentin Rothberg
+% April 2019
+
+# NAME
+containers-transports - description of supported transports for copying container images
+
+# DESCRIPTION
+
+**containers-storage:**_docker-reference_
+
+An image located in a local containers storage image store.
+Please refer to containers-storage.conf(5) for further information.
+
+**dir:**_path_
+
+An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files.
+This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
+
+**docker://**_docker-reference_
```

Comment:
_docker-reference_ may not include both a tag and a digest.

(Separately, _somewhere_ this should include an explanation of the reference format, incl. the `docker.io[/library]` canonicalization and the hostname vs. namespace in docker.io heuristic.)

Reply:
Done.